src/server/pkg/storage/chunk/storage.go

```diff
@@ -6,17 +6,21 @@ import (
 	"github.com/pachyderm/pachyderm/src/server/pkg/obj"
 )
 
+const (
+	// Prefix is the chunk storage prefix.
+	Prefix = "chunks"
+)
+
 // Storage is the abstraction that manages chunk storage.
 type Storage struct {
-	objC   obj.Client
-	prefix string
+	// (bryce) should we have a ctx here?
```

Comment:
I would guess not, it's probably better for context to be passed down from whatever operation is calling the API. Since I think this struct is basically going to live for the lifetime of the program right?

Reply:
Yeah, I agree. I don't really know what I was thinking when I put that there.